modules/luci-mod-status/root/usr/share/rpcd/acl.d/luci-mod-status-index.json

```diff
@@ -3,6 +3,7 @@
 		"description": "Grant access to main status display",
 		"read": {
 			"file": {
+				"/etc/board.json": [ "read" ],
 				"/proc/sys/net/netfilter/nf_conntrack_count": [ "read" ],
 				"/proc/sys/net/netfilter/nf_conntrack_max": [ "read" ],
 				"/usr/lib/lua/luci/version.lua": [ "read" ],
```

Comment:
@jow- On current master this line is gone and there's the following now:

`"/www/luci-static/resources/view/status/include": [ "list" ]`